Documentation/Scripts/generateMdFiles.py

```diff
@@ -251,7 +249,7 @@ def SimpleRepl(match):
 RX2 = [
     # parameters - extract their type and whether mandatory or not.
     (re.compile(r"@RESTPARAM{(\s*[\w\-]*)\s*,\s*([\w\_\|-]*)\s*,\s*(required|optional)}"), r"* *\g<1>* (\g<3>):"),
-    (re.compile(r"@RESTALLBODYPARAM{(\s*[\w\-]*)\s*,\s*([\w\_\|-]*)\s*,\s*(required|optional)}"), r"\n**Post Body**\n\n *\g<1>* (\g<3>):"),
+    (re.compile(r"@RESTALLBODYPARAM{(\s*[\w\-]*)\s*,\s*([\w\_\|-]*)\s*,\s*(required|optional)}"), r"\n**Request Body** (\g<3>)\n\n"),
```

Comment:
The param `1` that is being dropped here used to be the Swagger name for the request body. Having that name as the "body parameter" name was highly confusing for users who might think they need to wrap the entire request body in another JSON object with a property of that name. It certainly confused me in Foxx before we decided to hide that implementation detail.